app/controllers/provider_interface/decisions_controller.rb

```diff
@@ -22,6 +22,7 @@ def new_offer
       @application_offer = MakeAnOffer.new(
         actor: current_provider_user,
         application_choice: @application_choice,
+        standard_conditions: ['Fitness to Teach check', 'Disclosure and Barring Service (DBS) check'],
```

Comment:
Seems reasonable but may I suggest a small refactoring. How about moving these pre-checked standard conditions to a constant within `MakeAnOffer`, which then becomes the source of truth for what conditions are considered standard, instead of defining this in the controller.

You could then change `MakeAnOffer#initialize` to use that constant as a default value instead of `nil` and drop this line from the controller. There may be some tests that fail as a result (_if_ we have system spec exercising making an offer with no conditions, for example), but it'd be interesting to see what breaks.

Reply:
@mnacos good point https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1480/commits/e854c7fe6d259dd7d461f2fa40225aa0fbb7ced6 makes those values defaults in the `MakeAnOffer` constructor as advised.